git-day-1/README.md

```diff
@@ -0,0 +1,330 @@
+
+
+
+ # Part 1
+
+
+This workshop starts with a recap based on this week's prep material...
+
+## Exploring Github repositories
+
+### Recap activity 🕹️
+
+- Trainees do this section in pairs
+- ⏲️ 10 mins
+
+Let's recap some of the things from earlier in the week.
+
+Visit the following repository on Github: https://github.com/CodeYourFuture/cyf-demo-repo
+
+Answer the following questions:
+
+
+a) How many commits are there in this project?
+b) Who committed on Oct 31, 2020?  
+c) What changes did Illictonion do in the commit titled "Revert changes accidentally pushed in the past"?  
+d) How many files were added in the first commit? What were the names of the files?  
+e) What is the hash for the first commit in the history?  
+
+In your pair, discuss briefly:
+
+- what is a commit ?
+- why do we store repositories on Github ?
+- what is VSCode used for ?
+
+### 🧵🗣️ Whole group discussion 
+
+- Whole class discussion
+- ⏲️ 10 mins
+- A volunteer will need to facilitate this section
+
+A volunteer can facilitate a whole group discussion by going through the questions above and asking pairs for their feedback. If pairs are unsure / not quite accurate then spend a small amount of time addressing misconceptions. Remember to ask around for feedback 
+
+
+## Creating a fork 🍴
+
+### 🕹️ Recap activity 
+
+- Trainees do this section in pairs
+- ⏲️ 5 mins
+
+Use the guidelines from the prep section of this week to create a fork.
+
+1. On one person's computer, fork this repo: https://github.com/CodeYourFuture/cyf-demo-repo
+2. 📝 Double-check the URL of your forked repo. How can you tell the fork was successfully created?
+
+### Whole group discussion/demo 🧵 🗣️
+
+- Whole class discussion
+- ⏲️ 5 mins
+- A volunteer will need to facilitate this section
+
+A volunteer will need to navigate and let the group guide them in creating a fork as per the steps in the previous section.
+
+## 🌀 Cloning a repository 
+
+### 🕹️ Recap activity 
+
+- Trainees do this section in pairs
+- ⏲️ 5 mins
+
+#### 📋 Check your understanding
+
+Before continuing, try answering the following:
+
+❓ what is the difference between a **fork** and a **clone**.
+
+Remember to check your answer before continuing.
+
+
+1. Get a local copy of a repository on your local machine.
+2. Open this local repository using VSCode.
+3. Use `pwd` in your terminal to check you're in the right place.
+
+Now clone your forked repository from Github onto your local machine.
```

Comment:
Sorry for being late to the party but I'm struggling to make sense of this exercise. I'm happy to submit a PR with fixes but I wanted to understand the intention first.

> Get a local copy of a repository on your local machine.

"Get" is a bit ambiguous here: does this mean to open an existing repo? Or to clone a repo?

If it's the latter, which repo should they clone? Any repo? I think I would prefer a specific repo, as it'll just slow down the exercise to pick a repo to clone.

> Now clone your forked repository from Github onto your local machine.

Assuming the points above are to clone a forked repo, this is just repeating the same point?

Or is this line just in the wrong place - should it be above the list?